python/cudf/tests/test_feather.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+import cudf
+from cudf.tests.utils import assert_eq
+
+import pytest
+import os
+import pandas as pd
+import numpy as np
+import pyarrow as pa
+from string import ascii_letters
+from distutils.version import LooseVersion
+
+
+if LooseVersion(pd.__version__) < LooseVersion('0.24'):
+    try:
+        import feather # noqa F401
+        _have_feather = True
+    except ImportError:
+        _have_feather = False
+    if not _have_feather:
```

Comment:
Why the flag instead of calling the pytest.skip in the except?

Reply:
Good point, will update.